src/lib/metadata.ts

```diff
@@ -0,0 +1,70 @@
+import { MetadataEntry, EncryptedMetadataManager } from "@saleor/app-sdk/settings-manager";
+import { Client } from "urql";
+
+import {
+  FetchAppDetailsDocument,
+  FetchAppDetailsQuery,
+  UpdateAppMetadataDocument,
+} from "../../generated/graphql";
+
+export async function fetchAllMetadata(client: Client): Promise<MetadataEntry[]> {
```

Comment:
Since this is example, maybe we can add some JSDocs comments to treat this as "living documentation"?

I'm not saying this function is required to be commented, but maybe we can try to "enrich" example with additional comments, resources, why its like that etc. 

We can send example to somebody (please do!) and ask them if this is self explaining to them